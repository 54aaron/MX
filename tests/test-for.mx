int main() {
    int i;
    prints("==============================================");
    prints("for(i = 0; i < 10; i = i + 1)");
    for(i = 0; i < 10; i = i + 1) {
        if(i == 5) {
            prints("i = 5");
        } else {
            print(i);
        }
    }
    prints("==============================================");
    prints("for( i = 5; i > 0; i = i -1)");
    for( i = 5; i > 0; i = i -1) {
        print(i);
    }  
    prints("==============================================");
}
