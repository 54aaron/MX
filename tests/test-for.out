==============================================
for(i = 0; i < 10; i = i + 1)
0
1
2
3
4
i = 5
6
7
8
9
==============================================
for( i = 5; i > 0; i = i -1)
5
4
3
2
1
==============================================
